doc/_static/custom_styles.css

```
@@ -1,3 +1,8 @@
+/* Correctly wrap paragraphs*/
+p {
+  text-align: justify;
+}
+
```

```suggestion

.card-text {
    text-align: center;
}

```

This will keep the card text centered.

---

I think it works here @ccordoba12. If you agree with @CAM-Gerlach suggestion, @dalthviz, can you commit it please?